packages/shared/src/utils/contract/contract-types.ts

```diff
@@ -0,0 +1,77 @@
+import { ContractUpdate } from '@deriv/api-types';
```

Comment:
You don't have to review this file! I just merged it with my branch to fix TS errors for helpers